docs/DUAL_BAND.md

```diff
@@ -0,0 +1,19 @@
+# mLRS Documentation: Dual Band #
+
+([back to main page](../README.md))
+
+**Important: Dual band functionality is experimental and currently only available in the branch 'dev-dualband2'. Firmware binaries need to be compiled manually for the specific hardware.**
+
+mLRS allows for a dual-band setup, where the Tx module and receiver both transmit and receive in the 2.4 GHz and 868/915 MHz (or 433 MHz/70 cm) bands simultaneously.
+
+This operation mode is very simlar to a full-diversity setup, except that the hardware of course needs to provide two RF chains operating in the two different RF bands. An example is given by the dual-band version of the easy-to-solder boards, which can be eqipped with E77 modules working at 868/915 MHz and E28 modules working at 2.4 GHz. See [here]( https://github.com/olliw42/mLRS-hardware/tree/master/olliw-stm32-based/rx-tx-E77-E28-dualband-easysolder). Like full diversity, dual band also uses FHSS, and receiving depends on which packet in which band is best according to a full-diversity principle.
```

Comment:
Couple notes:

- Your call but I see both 'full-diversity' and 'full diversity' (with and without hyphen)
- For the last sentence, I think the want to eliminate the note about FHSS (as non-diversity setups also use FHSS), maybe 'Similar to a full-diversity setup, the receiver will choose the packet on the best band.'  
- Question, is RSSI used to determine which band is picked?  Or something else?

Reply:
1. I follow what some native speaker did once tell me: two words then word1 word2, three words then word1-word2 word3. So "full diversity" but "full-diversity setup". But I will follow your advice.
2. I did add it since one might think that with two rf bands it might just do it by sticking to a frequency in each band, but is doing each it's FHSS in each band. I thought this could be noteworthy.
3. We do have a crc to check if a packet is good, so we just pick a good one. So the combinations are good-good, good-bad, bad-good, bad-bad.

Reply:
1. I see the pattern now, I think it's fine to keep as is :)
2. Rework to include FHSS: 'Similar to a full-diversity setup, dual band uses FHSS and the receiver will choose the packet on the best band.'
3. Okay, good to know :)

Reply:
fantastic, many thx sir. Done.